pkg/config/config.go

```diff
@@ -365,6 +365,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("external_metrics.aggregator", "avg")                    // aggregator used for the external metrics. Choose from [avg,sum,max,min]
 	config.BindEnvAndSetDefault("external_metrics_provider.bucket_size", 60*5)           // Window to query to get the metric from Datadog.
 	config.BindEnvAndSetDefault("external_metrics_provider.rollup", 30)                  // Bucket size to circumvent time aggregation side effects.
+	config.BindEnvAndSetDefault("kubernetes_event_collection_timeout", 100)              // value in milliseconds
```

Comment:
I think you need to update the `https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config_template.yaml` with the new parameter

Reply:
yes, good catch!